projects/plugins/jetpack/3rd-party/jetpack-backup.php

```diff
@@ -0,0 +1,132 @@
+<?php
+/**
+ * Compatibility functions for the Jetpack Backup plugin.
+ * https://wordpress.org/plugins/jetpack-backup/
+ *
+ * @since 10.4
+ *
+ * @package automattic/jetpack
+ */
+
+namespace Automattic\Jetpack\Jetpack_Backup;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+const PLUGIN_SLUG = 'jetpack-backup';
+const PLUGIN_FILE = 'jetpack-backup/jetpack-backup.php';
+
+add_action( 'admin_notices', __NAMESPACE__ . '\error_notice' );
+add_action( 'admin_init', __NAMESPACE__ . '\try_install' );
+add_action( 'jetpack_activated_plugin', __NAMESPACE__ . '\configure_plugin', 10, 2 );
+
+/**
+ * Verify the intent to install Jetpack Backup, and kick off installation.
+ *
+ * This works in tandem with a JITM set up in the JITM package.
+ */
+function try_install() {
+	if ( ! isset( $_GET['jetpack-backup-action'] ) ) {
+		return;
+	}
+
+	check_admin_referer( 'jetpack-backup-install' );
+
+	$result = false;
+	// If the plugin install fails, redirect to plugin install page pre-populated with jetpack-backup search term.
+	$redirect_on_error = admin_url( 'plugin-install.php?s=jetpack-backup&tab=search&type=term' );
+
+	// Attempt to install and activate the plugin.
+	if ( current_user_can( 'activate_plugins' ) ) {
+		switch ( $_GET['jetpack-backup-action'] ) {
+			case 'install':
+				$result = install_and_activate();
+				break;
+			case 'activate':
+				$result = activate();
+				break;
+		}
+	}
+
+	$jetpack = \Jetpack::init();
+	if ( $result ) {
+		/** This action is already documented in _inc/lib/class.core-rest-api-endpoints.php */
+		do_action( 'jetpack_activated_plugin', PLUGIN_FILE, 'jitm' );
+		$jetpack->stat( 'jitm-plugin-install', 'jetpack-backup-success' );
```

Comment:
Should we aim to include this directly in the `Jetpack_Plugins` library, so this can benefit other plugins in the future?

Reply:
That sounds like a nice approach. I'll remove the stat entirely from this PR and add it separately.